GVFS/GVFS.UnitTests.Windows/Windows/Upgrader/UpgradeTests.cs

```diff
@@ -35,6 +35,63 @@ public virtual void Setup()
             this.Upgrader.LocalRingConfig = ProductUpgrader.RingType.Slow;
         }
 
+        public virtual void NoneLocalRing()
+        {
+            string errorString = "Upgrade ring set to None. No upgrade check was performed.";
+            this.ConfigureRunAndVerify(
+                configure: () =>
+                {
+                    this.Upgrader.LocalRingConfig = ProductUpgrader.RingType.None;
+                },
+                expectedReturn: ReturnCode.GenericError,

```

Comment:
Yea I don't know if this should be an error. If we think of "None" as effectively being an empty upgrade ring, then you're just always up to date and that's a success for the other two rings. @jrbriggs do you have an opinion on that?